layouts/partials/textprocessing.html

```diff
@@ -55,4 +55,29 @@
 {{/* Add jumpable anchors */}}
 {{ $content = $content | replaceRE "(<h[1-9] id=\"([^\"]+)\">)(.+)(</h[1-9]>)" `<a href="#${2}">${1}<span class="hanchor" ariaLabel="Anchor"># </span>${3}${4}</a>` }}
 
+{{/* Callouts */}}
+{{ $content = $content | replaceRE "<blockquote>" "<blockquote class=callout>" }}
```

Comment:
is the styling for `.callout` supposed to be empty? I don't see the styling for it

Reply:
The callout class is added to all `<blockquote>` tags and then removed from all non-callout blocks after adding the classes for the callouts. There will never be a blockquote with the `callout` class, just `*-callout` (like `note-callout`, etc.)

Reply:
Is there styling for the `*-callout` classes?

Reply:
Yeah, noticed that just now. Forgot to add it. should be fixed now.